doc improvement: move wifi fw page (#14273)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Wi-Fi
-----

* Added:

  * The :kconfig:option:`NRF_WIFI_FW_PATCH_DFU` Kconfig option to enable DFU support for nRF70 Series devices.
    This allows firmware patches for signed images to be sent to multi-image targets over Wi-Fi.

>   * The :ref:`ug_nrf70_fw_patch_update` user guide in the :ref:`ug_nrf70_developing` section that describes how to perform a DFU for firmware patches using Wi-Fi.

Reviewer: I would suggest to move line 224 under Documentation section after line 1367, as this doc is not added under Protocols section.

Wi-Fi
-----

> * Added the :kconfig:option:`NRF_WIFI_FW_PATCH_DFU` Kconfig option to enable DFU support for nRF70 Series devices.
>     This allows firmware patches for signed images to be sent to multi-image targets over Wi-Fi.

Reviewer 2: ```suggestion
* Added the :kconfig:option:`NRF_WIFI_FW_PATCH_DFU` Kconfig option that enables DFU support for nRF70 Series devices.
  This allows firmware patches for signed images to be sent to multi-image targets over Wi-Fi.
```

  * Support for Bluetooth LE Security Manager Protocol that allows to establish secure session with bridged Bluetooth LE devices.
  * Callback to indicate the current state of Bluetooth LE Connectivity Manager.
    The current state is shown by LED 2.

>   * Support for Wi-Fi firmware upgrades on external memory, for the nRF5340 DK with the nRF7002 EK attached.

Reviewer 3: ```suggestion
    * Support for the nRF5340 DK with the nRF7002 EK attached.
    * Support for Wi-Fi firmware upgrades on external memory, for the nRF5340 DK with the nRF7002 EK attached.
```

  * :ref:`ug_matter_configuring_read_client` in most Matter samples using the new :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option.
  * WPA Supplicant advanced features in all Matter samples using the :kconfig:option:`WPA_SUPP_ADVANCED_FEATURES` Kconfig option.
    This saves roughly 25 kB of FLASH memory for firmware images with Wi-Fi support.

* :ref:`matter_lock_sample` sample:

>   * Added support for Wi-Fi firmware upgrade on external memory.

Reviewer 3: This is valid only for `nrf5340dk_nrf5340 + nrf7002EK`, so could you mention it somewhere?

The second question, could you please add a similar note to `Matter Bridge` Application? Regarding that application, I've added support for `nrf5340dk_nrf5340 + nrf7002EK` (Previously we had support for `nrf7002dk_nrf5340` only), and now we have the same support for Wi-Fi firmware upgrade on external memory there (also valid for  `nrf5340dk_nrf5340 + nrf7002EK` only )

Author: Added, please confirm if it is correctly described.

* Disabled:

  * :ref:`ug_matter_configuring_read_client` in most Matter samples using the new :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option.
  * WPA supplicant advanced features in all Matter samples using the :kconfig:option:`WPA_SUPP_ADVANCED_FEATURES` Kconfig option.

>     This saves roughly 25 kB of FLASH memory for firmware images with Wi-Fi support.

Reviewer 2: ```suggestion
    This saves roughly 25 KB of flash memory for firmware images with Wi-Fi support.
```
@kittydepa tbc - don't think I saw this capitalized anywhere else
